modules/test/playwright/tests/workspaces/liferay-workspace-marketplace/marketplace-publisher-dashboard/marketplaceAppPublisher.spec.ts

```diff
@@ -0,0 +1,128 @@
+import {expect, mergeTests} from '@playwright/test';
+
+import {dataApiHelpersTest} from '../../../../fixtures/dataApiHelpersTest';
+import {getRandomInt} from '../../../../utils/getRandomInt';
+import {marketplacePagesTest} from '../fixtures/marketplacePages';
+import {marketplaceSiteFixture} from '../fixtures/marketplaceSite';
+import {PublishProductPayload} from '../types';
+import {products} from '../utils/constants';
+
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
```

Comment:
@welitonpe this license should be displayed on the top 

can you fix please? after that you can forward the pr